docs/src/layouts/MainLayout.vue

```diff
@@ -161,4 +176,75 @@ $adjust-header-viewport: 860px;
   z-index: 5; // ensures it's lower than header but higher than components page filter section
 }
 
+@function generateRandomStars($number-of-stars) {
+  $value: '#{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  @for $i from 1 through $number-of-stars {
+    $value: '#{$value}, #{random(7000)}px #{random(7000)}px #{$lp-primary}';
+  }
+  @return unquote($value);
```

Comment:
I'm confused, why is `unquote` needed?

Reply:
$value returns a quoted string, and unquote removes the quotes which are not needed in the final css